src/client/components/configuration/view.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react'
+import * as style from './style.css'
+import { Node, Tree } from './tree/tree'
+import { TreeStore } from './tree_store'
+
+const store = new TreeStore({
```

Comment:
I think its best that we stop using the word `store` and `model` and just use one through throughout the code base. Then we can maybe think of it as a more traditional "model view controller" architecture. So I suggest we replace every occurrence of `store` with `model`.